CHANGELOG.md

```diff
@@ -9,19 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Add test for autoscaling MachinePool nodes.
```

Comment:
I don't think these lines should have been removed, right?

Reply:
yeah